lib/ansible/config/ansible_builtin_runtime.yml

```diff
@@ -7625,8 +7627,8 @@ plugin_routing:
       redirect: community.general.hwc_utils
     ibm_sa_utils:
       redirect: community.general.ibm_sa_utils
-    keycloak:
-      redirect: community.general.keycloak
+    identity.keycloak:
```

Comment:
@felixfontein Is this right?

Reply:
Well, it's less wrong than before :) I'm not sure what the final correct syntax will be - I *think* for the key it is correct, but the `redirect` value needs to be improved. I guess @nitzmahone can better comment on this since he's working on fixing this in general.